### ISSUE TITLE
Not able to sweep limbo funds from closed channel after restoring wallet. (Android)

### ISSUE THREAD

**Issue by enoughofme** [lightninglabs/lightning-app#1244]
Tl;dr at bottom of description

### Description
Autopilot opened channel with all available funds; 1.6m Sats. Closing the channel resulted in unresponsive partner and automatic force close.

During force close, I had to restore wallet with seed. After restoring wallet the previous transactions show my initial deposits: id 06b85d0b146d877caf47f7a939a7d55ff637a845f0d59176bb8ec37c030f5fb7 and 4598be9fded7802e7c7b5d534a79e44625affcd2e5828144bead3f26a190c91e as confirmed and the channel opening transaction: id
511d526a692c521db69e20d98cfa4f7c8eb689136bec19f5cceeb90aa6164b5c shows as confirmed. But no balance or ability to find or sweep limbo balance from closed channel: transaction id
9c7b968065da6596f0d6ca265eb7a89c1a89ff9bee4862899ca7b33c79229c1a
wallet containing limbo balance:
bc1qphf8cr8zg9lhns28h4qjc3dfac2nupuauj23apmg867l06k8m02qdqtwt7 . I don't have the logs going back far enough, I assume this is because of recovering the wallet, maybe someone can tell me if there is a way to access them.

I am novice at best with Lightning but am eager to learn. I'm willing to accept if the funds are lost but I just want to understand the process and if I can do something to recover them, why it happened, what I could've done differently to avoid, and maybe if there is somewhere else I need to go to ask these questions, if not here. I love this product, Bitcoin and Lightning, but sadly I'm not as smart as a lot of people in the space. Though I'm probably a lot more capable than the average person, I hope to be a bridge between the highly capable and the layman so that these products can become more accessible for the general public.

Thank you for any advice, help, or constructive criticism anyone has to share.

Tl;dr I'm a moron who deposited too much money into app with autopilot enabled, closed channel, had to restore wallet with seed, now balance is 0, which put a fire under me to actually learn how lightning works in depth, which I should've done in the first place.

### Steps to reproduce the behavior

Download app
Create wallet
Wait for autopilot to open channel
Close channel
Have other party be unresponsive
Have App automatically force close channel
Restore wallet with seed within 24 hours
### Expected behavior
Force closed channel restores funds to wallet in 24 hours as stated in force close massage
### Actual behavior
No returned funds 4 days later. No information in app about sweeping limbo funds

**Comment by maltokyo** [comment]
I have the same issue as you. You’re not below average or stupid as you say. The technology isn’t there yet and is not transparent how funds can be recovered in case you restore from the word seed. I also restored (as no channels opened at all and I couldn’t even open my own manually, the errors that happened didn’t make any sense at all) and thought of I restore from seed everything may reset, and I will be able to open channels.  Restore worked fine but all my on-chain balance (I had no lightning balance at all, as it was “in progress” for more than a week) disappeared.  Watching this space in case someone else can help.

**Comment by molxyz** [comment]
@enoughofme  You made a mistake here:

> During force close, I had to restore wallet with seed.

You didn't have to restore the wallet with seed because of a force closing, all you needed to do was restart the app or restart the phone.
The seed can only restore the onchain fund that is already in your wallet, not fund from channels whether the channels are still opened or closed but not finalized onchain. 
Currently the app doesn't have `channel.backup` like the commandline fullnode LND so I don't know if there's another way to restore this fund for you, I doubt it.

**Comment by maltokyo** [comment]
> Currently the app doesn't have `channel.backup` like the commandline fullnode LND so I don't know if there's another way to restore this fund for you, I doubt it.

Wow...  Its exactly this type of warning that should be in the app home screen and installation process.  It was not clear for me that the seed is ONLY backling up the on-chain funds.  Crazy

**Comment by molxyz** [comment]
@maltokyo
> Restore worked fine but all my on-chain balance (I had no lightning balance at all, as it was “in progress” for more than a week) disappeared.

I'm confused by your statement. How did you restore your wallet? Were you using the Android app or Desktop app?

**Comment by maltokyo** [comment]
iOS app
